_search-plugins/search-pipelines/normalization-processor.md

```diff
@@ -74,6 +77,41 @@ PUT /_search/pipeline/nlp-search-pipeline
 ```
 {% include copy-curl.html %}
 
+The next example demonstrates how to use the `lower_bounds` parameter with the `min_max` normalization technique. It also omits the `weights` parameter in the combination technique, which will result in equal weighting of the queries by default:
+
+```json
+PUT /_search/pipeline/nlp-search-pipeline
+{
+  "description": "Post processor for hybrid search",
+  "phase_results_processors": [
+    {
+      "normalization-processor": {
+        "normalization": {
+          "technique": "min_max",
+          "parameters": {
+            "lower_bounds": [
+                {
+                  "mode": "apply",
+                  "min_score": 0.5
+                },
+                {
+                  "mode": "ignore"
+                }
+              ]
+          }
+        },
+        "combination": {
+          "technique": "arithmetic_mean"
+        }
+      }
+    }
+  ]
+}
+```
+{% include copy-curl.html %}
+
+In this example, the `lower_bounds` parameter is used to set different lower bounds for each query in a hybrid search scenario. For the first query, a lower bound of 0.5 is applied, while for the second query, the lower bound is ignored. This allows for fine-tuning of the normalization process for each individual query in a hybrid search.
```

Comment:
Can we also have show some calculation here with lower bound?

Reply:
added formulas in the section where `mode` param is described, I think that's the better fit